jboss/container/eap/cd/CD18/module.yaml

```diff
@@ -0,0 +1,15 @@
+schema_version: 1
+name: jboss.container.eap.cd
+version: 'CD18'
```

Comment:
the CD is implicit in the path itself, anyway, If we are going to use CDxx as a common pattern to define the release/version, then I would suggest changing https://github.com/wildfly/temp-eap-modules/tree/EAP7-1216/jboss/container/launch/cd/18.0 and https://github.com/wildfly/temp-eap-modules/tree/EAP7-1216/jboss/container/config/cd/17.0 according. Indeed  config/cd/17.0 should be config/cd/18.0 or config/cd/CD18.0

Reply:
+1 for @yersan's comment. I think 18.0 is probably more accurate -- you might end up with an 18.1 or something (unusual for CD, but common in the EAP case.)